repeated_test~

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
Please remove this file.

Reply:
Sorry, this was an Emacs temporary file so I missed this. I'm going to go ahead and add another `.gitignore` line for these types of files

Reply:
IMO this is the sort of thing that should go in your global __.gitignore__ not in the __.gitignore__ for Mesh. This is specific to your development setup, and it is not possible/realistic for us to account for every possible development environment in Mesh's __.gitignore__.